tcl/board/snps_hs5_fpga.cfg

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+#  Copyright (C) 2022 Synopsys, Inc.
+#  Shahab Vahedi <shahab@synopsys.com>
+
+#
+# HS Development Kit SoC.
+#
+# Contains a single ARC HS5 core.
+#
+
+adapter driver ftdi
+
+#source [find interface/ftdi/digilent-hs2.cfg]
```

Comment:
It didn't work because that file sets the device_desc of the ftdi device to the string "Diligent Adept USB Device", whereas the HAPS system provides "Diligent USB Device". In absence of a perfect match, the MPSSE subsystem refuses to proceed.

Let's keep it this way, as I think the alternative solution (including the file and then overwriting device_desc) is too unwieldy.